src/java.base/share/classes/java/lang/Exception.java

```diff
@@ -56,7 +57,7 @@
      * call to {@link #initCause}.
      */
     @SideEffectFree
-    public Exception() {
+    public @Unique Exception() {
```

Comment:
There are other constructors for this class. Shouldn't they also return a `@Unique` object?
Can you just look through the whole class bodies for the two methods and see whether other annotations are missing?

Reply:
I have annotated the remaining constructors as @Unique in my lates commit.